index.html

```diff
@@ -2806,216 +2799,180 @@ <h2>CBOR</h2>
+<li>Other Data Types not otherewise defined in the Abstract Data Model MUST be represented as a <a href="https://tools.ietf.org/html/rfc7049#section-2.1">CBOR String type. (major type 5)</a></li>
+<li>Indefinite-length items are not allowed and MUST be made <a href="https://tools.ietf.org/html/rfc7049#section-2.2.1">CBOR definite length.</a></li>
+<li>Datetime MUST be encoded as <a href="https://tools.ietf.org/html/rfc7049#section-2.4.1">CBOR UTF-8 (major type 3)</a>. This datetime value MUST be normalized to UTC 00:00, as indicated by the trailing "Z".</li>
+</ul>
+
+
+<!-- <p>Additional considerations to enable lossless production of a CBOR representation and to enable a deterministic 
+  canonical CBOR representation of the DID Document between other core representations of the DID document is possible 
+  with additional constraints to the CBOR representation.  -->
+  <p>
+  To produce a deterministic canonical CBOR representation of a DID document and faciliate maximal lossless compatiblity with other 
+  core representations via the Abstract Data Model the following constraints of a CBOR representation of a DID Document model MUST be followed:
+ 
+<ul>
+    <li>Property names MUST be represented as text string (major type 3) and contain only UTF-8 strings. </li>
+    <li>Undefined Values of Required Properties as defined in <a href="#data-model">the Data Model</a> that are absent from the CBOR representation SHOULD be labeled with Primitive type (major type 7) with value 23 (Undefined value).  </li>
+    <li>Property names in each CBOR map MUST be unique. </li>
+    <li>Integer encoding MUST be as short as possible.</li>
+    <li>The expression of lengths in CBOR major types 2 through 5 MUST be as short as possible. </li>
+    <li>The keys in every map must be sorted lowest value to highest. Sorting is performed on the bytes of the representation of the keys. If two keys have different lengths, the shorter one sorts earlier.  If two keys have the same length, the one with the lower value in (byte-wise) lexical order sorts earlier. </li>
+  </ul>
+</p>
 
-did = tstr .pcre "^did\\:(?&lt;method-name>[a-z0-9]{2,})\\:(?&lt;method-specific-id>[A-Za-z0-9\\.\\-\\:\\_]+)"
 
-did-url =  tstr .pcre "^did\\:(?&lt;method-name>[a-z0-9]{2,})\\:(?&lt;method-specific-id>[A-Za-z0-9\\.\\-\\:\\_]+)\\;(?&lt;path>[A-Za-z0-9\\/)(?&lt;query>\\?[a-z0-9\\=\\&amp;])#(?&lt;fragment>.+)"
+<p class="issue">
+ How to represent Floating-point values that can exceed the range or the precision IEEE 754. See <a href=https://github.com/w3c/did-core/issues/361">issue #361. </a>
+</p>
 
-service = {
-  id : did-url
-  type : text
-  serviceEndpoint : uri
-  ? description : text
-  * tstr => any
-}
-      </pre>
+<p>
+  The property name `@context` MAY be present in the CBOR representation of a DID Document and if present SHOULD be ignored as this property is reserved for JSON-LD processing.
+</p>
```

Comment:
This block should be removed as a similar statement is removed from the JSON section in another PR and we shouldn't add them. If anything, this should just be a note that it might show up, but no normative language here as it's not a defined property.

Reply:
Agreed, the Editors have discussed items such as this in the PR and will clean it up in another PR given how long this PR has been hanging out there. At this point, it's an improvement on what is in the specification and the Editors will do their best to align it with the rest of the specification while giving folks the opportunity to object/revise in a future PR.